Polyhedron/demo/Polyhedron/Scene_polyhedron_item.h

```diff
@@ -24,6 +24,7 @@ class SCENE_POLYHEDRON_ITEM_EXPORT Scene_polyhedron_item
         : public CGAL::Three::Scene_item{
     Q_OBJECT
 public:
+  typedef Polyhedron FaceGraph;
     enum STATS {
       NB_VERTICES = 0,
```

Comment:
Ther eis already the polyhedron() function that does that. In surface_mesh, the mesh is called SMesh, not Polyhedron, hence the need of this typedef, but polyhedron() exists for both, so there shouldn't be a need for a function called face_graph().